src/components/modes/teacher/Settings.js

```diff
@@ -180,6 +205,17 @@ class Settings extends Component {
           )}
           label={t('Show Time Controls to Students')}
         />
+        <Tooltip title="Starting automatically will remove student controls">
```

Comment:
Remember to change the tooltip title:

"If the timer does not start automatically and there are no controls, the student will still be able to see a start button."

Also, wrap in `t()` function.